dashboard/src/components/ServiceInstanceView/ServiceInstanceView.test.tsx

```diff
@@ -208,5 +230,32 @@ context("when all the components are loaded", () => {
       ).toExist();
       expect(wrapper).toMatchSnapshot();
     });
+
+    it("should not show bindings information if the class is not bindable", () => {
+      const classes = {
```

Comment:
Maybe you can share these `classes` in the tests and only override the `spec.bindable` value during mount?